wdl/wavgen.py

```diff
@@ -1062,6 +1064,8 @@ def plot(self, cycles=2, initialLevel=None):
         #               Tracer()()
         return
 
+def state_empty(sstr):
```

Comment:
suggest we start modernising just a little. This should work with anything > python 3.5

```def state_empty(sstr: str) -> bool:```

etc